app/controllers/application_controller/feature_flags_dependency.rb

```diff
@@ -37,4 +37,9 @@ def classroom_visibility_enabled?
     logged_in? && current_user.feature_enabled?(:classroom_visibility)
   end
   helper_method :classroom_visibility_enabled?
+
+  def redesign_enabled?
+    GitHubClassroom.flipper[:redesign].enabled? || (logged_in? && current_user.feature_enabled?(:redesign))
```

Comment:
Should we specify that this is an `onboarding_redesign` ? (unless we're covering more than that, just wanted to make it a little clearer what's involved)

Reply:
I'm down for updating. The original note created by @jeffrafter  said to create one called `redesign` so I just went with that, but, I definitely like greater specificity. 

@jeffrafter is it alright if we make the feature flag `onboarding_redesign`?

Reply:
We spoke offline and more specific is 👍